src/wallet/PegIn.cpp

```diff
@@ -0,0 +1,60 @@
+#include "PegIn.h"
+
+#include <mw/crypto/Blinds.h>
+#include <mw/wallet/Wallet.h>
+#include <mw/wallet/KernelFactory.h>
+#include <mw/wallet/OutputFactory.h>
+#include <mw/wallet/TxFactory.h>
+
+// TODO: Sort inputs, outputs, kernels, and owner_sigs
+mw::Transaction::CPtr PegIn::CreatePegInTx(const uint64_t amount, const StealthAddress& receiver_addr) const
+{
+    // Create peg-in output.
+    // We randomly generate the sender_key and output_blind.
+    // Receiver key is generated by OutputFactory using the wallet's stealth address.
+    SecretKey sender_key = m_wallet.NewKey();
+    BlindingFactor output_blind = Random().CSPRNG<32>();
```

Comment:
Wonder why its not Random::CSPRNG<32>() ? There's a few other occurences.

Reply:
It should be. I forgot it was static. I'll fix all instances of that.